docs/architecture/adr-043-interchain-identifiers.md

```diff
@@ -0,0 +1,108 @@
+# ADR 043: Interchain Identifiers
+
+## Context
+
+In the Internet of Blockchains We need a way to identify and locate assets within their chain namespace, using web-standard ([RFC39](https://datatracker.ietf.org/doc/html/rfc3986)) Universal Resource Indentifiers (URIs).
+
+When we refer to on-chain assets, this includes tokens, wallets, name records, or any other uniquely identifiable entity within a chain namespace.
+
+**Interchain Identifiers** (IIDs) are a standards-compliant method (based on [W3C DID-Core](https://w3c.github.io/did-core/)) for uniquely identifying, locating, referring to and verifying digital assets which are accessed within, or through, chain namespaces.
+
+IIDs also enable (off-chain) assertions to be made about (on-chain) digital assets – for instance, in the form of Verifiable Credentials.
+
+An IID Method DID is **self-referential**, which means the asset describes and locates itself in its chain namespace.
+
+Compare this with a generic, all-purpose DID which can be used to identify and refer to _any_ entity. For instance, the canonical application of DIDs is Self-Sovereign Identity, which provides people and other entities with identifiers together with cryptographic methods to authenticate as the subject and to exercise control over the state of the DID Document.
+
+---
+We define **Interchain Identifiers** as being explicitly purposed to identify and locate assets within a chain namespace.
```

Comment:
IIDs define a pattern of DID usage, so any method (on any chain) could be an IID method. It would be straightforward to set up an IID using Ethereum, or update an existing ethereum DID method to support it. Their current restrictions on property values means that did:ethr is NOT IID compatible (and they are intended as general purpose identifiers), but they have expressed some interest in supporting the features of IIDs; we'll be advancing that conversation once we get the new drafts ready.

I believe it is the chain agnostic aspect of IIDs that makes them so powerful. They can be used to identify and interact with NFTs on ANY chain. Once DApps and wallets start supporting IIDs, it will spur more chains to support IIDs, which will spur more DApps and wallets to do so, etc.